routes/index.js

```diff
@@ -61,6 +71,10 @@ router.get('/room/:name', function (req, res) {
   if (roomToSessionIdDictionary[roomName]) {
     sessionId = roomToSessionIdDictionary[roomName];
 
+    const tokenOptions = {};
+    // we need caption to be moderator role for captions to work
+    tokenOptions.role = "moderator";
+
```

Comment:
This can be moved up outside the if block. The `tokenOptions` is set the same in both cases